policy-test/tests/accepted_http_route_status.rs

```diff
@@ -121,7 +121,7 @@ async fn inbound_multiple_parents() {
         assert_eq!(invalid_cond.reason, "NoMatchingParent");
 
         // Find status for valid parent and extract the condition
-        let valid_cond = find_condition(parent_status.clone(), "test-valid-server")
+        let valid_cond = find_condition(parent_status, "test-valid-server")
```

Comment:
Sorry, this snuck in when I merged in the latest main. We're doing cloning here when we shouldn't. Thought I'd fix it as part of this. As a follow-up PR, I'll look to add some integration tests.